src/scielo/bin/xml/prodtools/xc.py

```diff
@@ -132,6 +133,16 @@ def call_convert_package(self, package_path):
         self.convert_package(package_path)
         return 'done', 'blue'
 
+    def _create_package_instance(self, source: str, output: str) -> SPPackage:
+        """Cria instância da classe SPPackage para o pacote de entrada"""
+
+        try:
+            package_maker = PackageMaker(source, output)
+            package = package_maker.pack()
+        except Exception as exc:
+            raise ScieloPackageError(str(exc))
```

Comment:
suspeito que deveria ser somente ScieloPackageError(exc)

Reply:
entendi... há um issue aberto só para tratar das exceções deste módulo: #3230. Então, poderia deixar para quem for resolver o ticket de dar o tratamento adequado.

Reply:
Alterei, obg ;)